.changelog/4553.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support for including release date alongside version number in sdk-changelog script.
```

Comment:
```suggestion
- description: Future SDK releases will include the release date.
```

Reply:
@dorschw @shmuel44 
This is still **not** one of the approved templates.
Why not:
```diff
+ - description: Added support for version number and release date when generating changelog entry for an SDK release.
```

Reply:
Done